views/admin/index.php

```diff
@@ -76,6 +75,17 @@
           },
         ],
         [
+            'attribute' => 'auth_items',
+            'format' => 'html',
+            'value' => function ($data) {
+                return implode(', ', array_map(function ($ai) {
+                    return $ai->name;
+                }, $data->authItems));
+            },
+            'filter' => ArrayHelper::map(Yii::$app->db->createCommand('select name from auth_item')->queryAll(), 'name', 'name'),
+            'visible' => Yii::$app->hasModule('rbac'),
```

Comment:
I think you should check something like `Yii::$app->get('authManager') instanceOf ManagerInterface`.